src/avram/validations.cr

```diff
@@ -143,11 +144,11 @@ module Avram::Validations
       size = attribute.value.to_s.size
 
       if !min.nil? && size < min
-        attribute.add_error "is too short"
+        attribute.add_error message
```

Comment:
These still need to have the same changes as you did below, then it will be good 👍